src/drivers/drv_sensor.h

```diff
@@ -182,6 +182,7 @@
 #define DRV_DIST_DEVTYPE_GY_US42       0x9C
 
 #define DRV_BAT_DEVTYPE_BATMON_SMBUS   0x9d
+#define DRV_POWER_DEVTYPE_INA23X       0x9E
```

Comment:
Is it easy to detect if the module is an INA237, INA238, or INA239? If so I would still use a unique id so that from logs we see what people actually have connected.

Reply:
I went a bit on a limb here - INA239 is actually a SPI device, so not supported. INA237 is a lower spec binned INA238, even says 238 in the DEVICE_ID register. Additionally, there is an INA236 which is different, so also not supported, while the name INA23X would imply that it was. Changed this driver to INA238 only - and by extension also supports INA237 since it's the same die.

Reply:
Made specific.